client/test/e2e/patient/invoice.spec.js

```diff
@@ -30,7 +30,7 @@ describe('patient invoice', function () {
     browser.get(path);
   });
 
-  it('invoices a patient for a single item', function () {
+  it.skip('invoices a patient for a single item', function () {
```

Comment:
Yes, the patient invoice module relies on `journal` to post, who's code I completely removed.  This skip was introduced last minute in https://github.com/IMA-WorldHealth/bhima-2.X/pull/320/commits/d1cab30054e375073e00dc9f84e5c052a75e4d42. Whoever works on the patient invoice module will be responsible for implementing this route and unskipping the test (as well as possibly writing a few more) to prove that this whole procedure works.